extensions/2.0/Khronos/KHR_materials_specular/README.md

```diff
@@ -0,0 +1,100 @@
+# KHR\_materials\_specular
+
+## Khronos 3D Formats Working Group
+
+* TODO
+
+## Acknowledgments
+
+* TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds two parameters to the metallic-roughness material: `specular` and `specularColor`.
+
+`specular` allows users to configure the strength of the specular reflection in the dielectric BRDF. A value of zero disables the specular reflection, resulting in a pure diffuse material. The metal BRDF is not affected by the parameter.
+
+`specularColor` changes the F0 color of the specular reflection in the dielectric BRDF, allowing artists to use effects known from the specular-glossiness material (`KHR_materials_pbrSpecularGlossiness`) in the metallic-roughness material.
```

Comment:
I was under the impression that _we_ (Khronos) should strive at moving away from the specularglossiness workflow and focus on metallicroughness going forward.
It's not clear to me if this is a copy of an effect achieved in the specgloss workflow or if it is a new parameter in the metalrough workflow.
Maybe I am misunderstanding......